test/Makefile.in

```diff
@@ -51,7 +51,7 @@ endif
 
 DMTCP_INCLUDE=${top_srcdir}/include
 
-CFLAGS += -I${DMTCP_INCLUDE} -DDMTCP -fPIC -g
+CFLAGS += -I${DMTCP_INCLUDE} -DDMTCP -fPIC -g -std=gnu99
```

Comment:
Because we want to use some modern features like inline for loop initializer, etc. This should have been in there all along :).